### ISSUE TITLE
Parallelize SOM-Toolbox

### ISSUE THREAD

**Issue by jiperez91** [ilarinieminen/SOM-Toolbox#17]
Hi everyone:

There is a way to parallelize the som_make or som_batchtrain functions? I want to speed up this functions but I don't have the Parallel Computing Toolbox.

**Comment by twocs** [comment]
I don't believe that it would be straightforward to parallelize the functions in SOM-Toolbox. It would basically require line-by-line modifications. 

But have you checked out the C++ project Somoclu.
"Somoclu is a massively parallel implementation of self-organizing maps" https://github.com/peterwittek/somoclu

Somoclu has different interfaces, including Python, R and MATLAB version. In its Readme, it states that "the MATLAB version [integrates] with somtoolbox." That means you can prepare the SOM in Somoclu and still use SOM-Toolbox for visualization.
